src/vivarium_census_prl_synth_pop/components/synthetic_pii.py

```diff
@@ -31,7 +31,7 @@ class SSNGenerator(GenericGenerator):
     def name(self):
         return "SSNGenerator"
 
-    def generate(self, df_in: pd.DataFrame) -> pd.DataFrame:
+    def generate(self, df_in: pd.DataFrame, additional_key: Any = None) -> pd.DataFrame:
         """Generate synthetic Social Security Numbers
 
         Parameters
```

Comment:
might be nice to add to the "results" here the guarantee that they are unique for the inputted df_in